internal/cli/up.go

```diff
@@ -203,3 +203,7 @@ func provideWebURL() (model.WebURL, error) {
 	}
 	return model.WebURL(*url), nil
 }
+
+func provideSyncletExecFlag() engine.ExecFlag {
```

Comment:
hmmm...can you move this provider into Engine? This should use the injected UpdateMode rather than reading directly from the flag